Math/src/main/java/org/tribuo/math/neighbour/bruteforce/NeighboursBruteForceFactory.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tribuo.math.neighbour.bruteforce;
+
+import com.oracle.labs.mlrg.olcut.config.Config;
+import org.tribuo.math.distance.DistanceType;
+import org.tribuo.math.la.SGDVector;
+import org.tribuo.math.neighbour.NeighboursQueryFactory;
+
+public final class NeighboursBruteForceFactory implements NeighboursQueryFactory {
```

Comment:
Could you add a little bit of javadoc for this class?

Reply:
I've added javadoc here and to `NeighboursQueryFactory` as well.